dpnp/dpnp_algo/dpnp_algo.pxd

```diff
@@ -211,17 +211,19 @@ cdef extern from "dpnp_iface.hpp":
 
 # C function pointer to the C library template functions
 ctypedef void(*fptr_1out_t)(void *, size_t)
-ctypedef void(*fptr_1in_1out_t)(void * , void * , size_t)
-ctypedef void(*fptr_2in_1out_t)(void * , void*, void*, size_t)
-ctypedef void(*fptr_2in_1out_new_t)(void * , void*, size_t, void*, size_t) # to be fused with fptr_2in_1out_t
-ctypedef void(*fptr_2in_1out_full_t)(void *, const void *, const size_t, const long*, const size_t,
-                                     const void *, const size_t, const long*, const size_t, const long*)
+ctypedef void(*fptr_1in_1out_t)(void *, void *, size_t)
+ctypedef void(*fptr_2in_1out_t)(void *, void *, void *, size_t)
+ # to be fused with fptr_2in_1out_t
+ctypedef void(*fptr_2in_1out_new_t)(void *, const void *, const size_t, const long *, const size_t,
```

Comment:
Do I understand correctly that this func point name `fptr_2in_1out_new_t` is temporary?

And in the future, you will delete the current `fptr_2in_1out_t` and rename `fptr_2in_1out_new_t` to `fptr_2in_1out_t`?

If not it is better rename `fptr_2in_1out_new_t`. 
`new` word may confuse.

Reply:
You are right. `fptr_2in_1out_new_t` will be renamed to `fptr_2in_1out_t` in further.